avocado/core/test.py

```diff
@@ -116,7 +116,7 @@ class Test(unittest.TestCase):
     default_params = {}
 
     def __init__(self, methodName='test', name=None, params=None,
-                 base_logdir=None, tag=None, job=None, runner_queue=None):
+                 base_logdir=None, job=None, runner_queue=None):
```

Comment:
Btw for the sake of keaping the interface, we can keep the API we can keep supporting this `tag` and only log warning that it should not be used as it does nothing... (we do have `self.warn` already so it's quite simple)

anyway I'd prefer removing it now as LTS is approaching. (or add the warning now and remove it in the next sprint as a compromise).